SDWebImage/SDAnimatedImageView.m

```diff
@@ -459,11 +459,13 @@ - (void)startAnimating
 - (void)stopAnimating
 {
     if (self.animatedImage) {
+        SD_LOCK(self.lock);
```

Comment:
```suggestion
[self.fetchQueue stopAnimation]; // Stop next frames in background
```

Reply:
In `resetAnimatedImage` method :
```
 [_fetchQueue cancelAllOperations];
```
So, It 's necessary ?

Reply:
Maybe put this cancel code into `stopAnimation` is better. Since it's the what we think `stop animation` should do.

This may reduce the progressive aniamted image rendering a little, since now we do not query next frame when user call `stopAnimation`. But put this decoding cost when new data buffer available. But it's OK for most cases.

Reply:
OK